item.rb

```diff
@@ -0,0 +1,26 @@
+require 'date'
+require_relative 'Method/genre'
+require_relative 'Method/label'
+
+class Item
+  attr_accessor :id, :genre, :author, :source, :label, :publish_date, :archived
+
+  def initialize(publish_date)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date #  user should enter a valid date [year, month, day]
+    @archived = false
+  end
+
+  def can_be_archived?
+    date = Date.new(@publish_date.to_i)
+    (Date.today - date).to_i > 3650
+  end
+
+  def move_to_archive
+    if can_be_archived? == true
+      @archived = true
+    else
+      puts 'Item cannot be archived'
+    end
+  end
+end
```

Comment:
Every item should have info about the `Genre`, `Author`, and `Label`. However, in your case, you have selected a generic class for each item. Eg., the `Book` class has info about `Label` and `Genre` but doesn't hold info about `Author`. Kindly fix this issue for all the Item classes.
I have recorded [this video](https://www.loom.com/share/f47a76f418374873bc5dbe41fecf1d17?sid=50059ae5-df2e-4456-acf1-d33542e0bdc7) where i explained in detail about this issue.

Reply:
Hello @muneebulrehman 

I hope you are doing well. I am thankful that you have done an in-depth review of our project. But we believe there are a few things that need your attention. 

- For team member 1, `Book` should have a book class and label class, not any other class like `genre` or `author`. I have attached a screenshot for your reference. 

![image](https://github.com/Booze33/Ruby-Capstone/assets/125204779/6a9e3708-1652-4866-9f86-41d8c0bf3d3f)

- For team member 2, `Music Album` should have a music_album class and genre class, not any other class like `lable` or `author`. I have attached a screenshot for your reference. 

![image](https://github.com/Booze33/Ruby-Capstone/assets/125204779/e77aecb3-edda-4b95-a1e7-5e96c9818a7d)

- For team member 3, `Game` should have a game class and author class, not any other class like `genre` or `lable`. I have attached a screenshot for your reference. 

![image](https://github.com/Booze33/Ruby-Capstone/assets/125204779/94081411-9a27-48b2-b8c1-615def5bfa9c)